crates/lang/src/env_access.rs

```diff
@@ -128,6 +156,12 @@ where
 
     /// Returns the price for the specified amount of gas.
     ///
+    /// # Example
+    ///
+    /// ```
+    /// let price = self.env().weight_to_fee(13);
```

Comment:
There is no difference between gas and weight. Please keep in mind that everyone else calls it gas, though. Removing the word "gas" would lead to less confusion in the future but more confusion right now.